dogm/demo/utils/CMakeLists.txt

```diff
@@ -28,4 +28,8 @@ TARGET_INCLUDE_DIRECTORIES(utils
     ${PROJECT_SOURCE_DIR}/include
     ${OpenCV_INCLUDE_DIRS})
 
-TARGET_LINK_LIBRARIES(utils PRIVATE ${OpenCV_LIBS})
\ No newline at end of file
+TARGET_LINK_LIBRARIES(utils PRIVATE ${OpenCV_LIBS})
+
+if(BUILD_TESTS)
```

Comment:
I think it would be good for this to be an option. This way it can be set in cmake-gui.

Reply:
Excellent idea! I set it to on per default.